docs/source/en/quantization/quant_bnb.md

```diff
@@ -0,0 +1,486 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Quantize Your Diffusion Models with `diffusers`
+
+Quantization reduces the number of bits used to represent numbers in a model's parameters,
+saving memory without significantly sacrificing accuracy. This can be a game-changer for diffusion
+models, which are popular for generating high-quality images but are notoriously resource-hungry.
+While these models run well on high-end setups or in the cloud, lighter versions are essential for
+consumer-grade devices—and that's where quantization comes into play.
+
+In this guide, we'll explore different types of quantization, from 8-bit to NF4—using `bitsandbytes` [1],
```

Comment:
Why don't just link instead?

Reply:
I also see that @stevhliu suggested the same, so will put the links as I go along.